autoheal-dev.yml

```diff
@@ -0,0 +1,119 @@
+#
+# Copyright (c) 2018 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This is an example configuration file used for development.
+
+#
+# This section contains the details needed to connect to the AWX or Ansible
```

Comment:
I'd suggest you remove all these duplicated comments from the `autoheal-dev.yml` file, otherwise they will eventually get out of sync. Remove also all the parameters that have default values that you don't need to change.

Reply:
:+1:  done